e2e-tests/resources/authd/utils.resource

```diff
@@ -0,0 +1,126 @@
+*** Settings ***
+Documentation       authd test resources
+
+Resource            kvm.resource
+Library             ./StringUtils.py    AS    StringUtils
+Library            ./VideoLogger.py    AS    VideoLogger
+Library             Process
+Library             OperatingSystem
+
+
+*** Keywords ***
+Log In
+    Match Text    Not listed    300
+    Hid.Keys Combo    Return
+    BuiltIn.Sleep    2
+    Hid.Type String    ubuntu
+    BuiltIn.Sleep    2
+    Hid.Keys Combo    Return
+    BuiltIn.Sleep    2
+    # Give some time for the desktop to settle
+    BuiltIn.Sleep    5
+    Hid.Move Pointer To Proportional    0    1
+    Match Text    Show Apps    300
+    Update System Time
+
+
+Log Out
+    Run Command    gnome-session-quit --logout --no-prompt
+
+
+Update System Time
+    Run Command In Terminal    sudo service chronyd restart && sudo chronyc waitsync
```

Comment:
Is chrony even installed? It's not installed by default on Ubuntu AFAICT, and I don't see it being installed explicitly, so if it installed I wonder which package pulls it in as a dependency.

Can't we just use systemd-timesyncd?

```suggestion
    Run Command In Terminal    sudo systemctl restart systemd-timesyncd.service
```

Reply:
Ah ok, I didn't know that. That seems like the right approach then. Let's just use the systemd executable instead of the sys V init one to restart the service:

```suggestion
    Run Command In Terminal    sudo systemctl restart chronyd && sudo chronyc waitsync
```